File: scripts/commit_carousel.py
import subprocess
import os

os.chdir('/vercel/share/v0-project')

try:
    subprocess.run(['git', 'add', 'app/page.tsx'], check=True, capture_output=True)
    subprocess.run(['git', 'commit', '-m', 'Update carousel: reduce to 8 banks, remove bank names'], check=True, capture_output=True)
    subprocess.run(['git', 'push', 'origin', 'update-location-text'], check=True, capture_output=True)
    print('Changes committed and pushed successfully')
except subprocess.CalledProcessError as e:
    print(f'Error: {e.stderr.decode()}')
    exit(1)

File: scripts/commit-carousel.js
import { execSync } from 'child_process';

try {
  execSync('git add -A', { cwd: '/vercel/share/v0-project', stdio: 'inherit' });
  execSync('git commit -m "Update carousel: reduce to 8 banks, remove bank names"', { cwd: '/vercel/share/v0-project', stdio: 'inherit' });
  execSync('git push origin update-location-text', { cwd: '/vercel/share/v0-project', stdio: 'inherit' });
  console.log('Changes committed and pushed successfully');
} catch (error) {
  console.error('Error:', error.message);
  process.exit(1);
}
